{"name": "test_Wijken_failure", "status": "passed", "description": "Failure API request for the Wijken", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/wijken/0363097000000C to validate whether the no record found with `failure code 404`", "status": "passed", "start": 1646842909080, "stop": 1646842909215}], "start": 1646842909080, "stop": 1646842909215, "uuid": "34b5c342-42d4-4c53-893e-60cc68eff4b2", "historyId": "037430a115a2552756c28c3061e688ed", "testCaseId": "42c285f4cbf7c1d3064bb577bde9fffe", "fullName": "e2etest.test_datasetgebiedenwijken#test_Wijken_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenwijken"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "59177-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenwijken"}]}